samples/core/parameterized_tfx_oss/README.md

```diff
@@ -25,6 +25,9 @@ The notebook sample includes the installation of various dependencies as its fir
 it depends on the latest released KFP and a nightly built TFX to leverage `TFX::RuntimeParameter`.
 
 # Permission
+
+> :warning: If you are using **full-scope** cluster in hosted pipeline beta version, **DO NOT** follow this section.
+
 This pipeline requires Google Cloud Storage permission to run. 
 If KFP was deployed through K8S marketplace, please follow instructions in 
 [the guideline](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials)
```

Comment:
and the  samples/core/parameterized_tfx_oss/taxi_pipeline_notebook.ipynb permission section which has the orignial bugbash concern

Reply:
Done. Update the section. Thanks!